code/modules/reagents/machinery/reagentgrinder.dm

```diff
@@ -1,5 +1,21 @@
-/obj/machinery/reagentgrinder
 
+
+var/global/list/juice_items = list (
```

Comment:
atomicity?

normally I wouldn't complain but I'm actually cleaning up grinder code at the moment (not that there's any way you could've known this)